### PR TITLE
Re-add umbrel compare

### PR DESCRIPTION
**Pull request by kn0wmad**

### CODE REVIEW

**Review by MattDHill — DISMISSED** [review]

**Comment by kn0wmad** [comment]
Added to StartOS FAQ.  In refactor, this will likely belong on the website FAQ.  Perhaps we might even like to have a comparison page following 040 that also compares projects like Synology.

File: site/source/support/faq/umbrel-compare.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+=================
+Umbrel Comparison
+=================
+
+.. contents::
+  :depth: 2 
+  :local:
+
+By popular demand, Start9 has provided a brief compare/contrast between StartOS and Umbrel.
```
Comment (MarnixCroes) [inline site/source/support/faq/umbrel-compare.rst:11]:
```suggestion
By popular demand, Start9 has provided a brief comparison/contrast between StartOS and Umbrel.
```

File: site/source/support/faq/umbrel-compare.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+========
+As of June 1, 2023, there are approximately 100 services available for Umbrel, and 40 services available for StartOS. Both offer developer docs that permit anyone anywhere to package anything, and neither is capable of gatekeeping.
+
+Architecture
+============
+Umbrel itself is an *app* that can be *installed* onto most Debian or Ubuntu computers and virtual machines. StartOS is an *operating system* (Linux distro) that can be *flashed* onto most computers and virtual machines.
+
+Config
+======
+Services often provide config files, such as Bitcoin's ``bitcoin.conf``. On Umbrel, updating config files requires using SSH and the command line. With StartOS, config files are presented as rich forms, including descriptions, downdowns, toggles, validated inputs, etc.
```
Comment (MarnixCroes) [inline site/source/support/faq/umbrel-compare.rst:34]:
```suggestion
Services often provide config files, such as Bitcoin's ``bitcoin.conf``. On Umbrel, updating config files requires using SSH and the command line. With StartOS, config files are presented as rich forms, including descriptions, dropdowns, toggles, validated inputs, etc.
```

Comment (MarnixCroes) [reply site/source/support/faq/umbrel-compare.rst:34]: I think you mean this?

**Review by MattDHill — APPROVED** [review]

**Pull request merged**